.github/workflows/checksum-test.yml

```diff
@@ -0,0 +1,29 @@
+name: Checksum Testing
+on: push
+
+jobs:
+  updateURLS:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout ide
+        uses: actions/checkout@v3
+      - name: Checkout ide-urls
+        uses: actions/checkout@v3
+        with:
+          repository: alfeilex/ide-urls-checksum
+          path: ide-urls
+          token: ${{ secrets.ACTION_PUSH_TOKEN }}
+      - name: Build and run url updater
+        run: |
+          cd url-updater
+          mvn install
+          mvn exec:java -Dexec.mainClass="com.devonfw.tools.ide.url.UpdateInitiator" -Dexec.args="../ide-urls"
+      - name: Commit and push to ide-urls
+        run: |
+          cd ide-urls
+          git checkout -b with-checksum-testing
+          git add .
+          git config --global user.name ${{ secrets.BUILD_USER }}
+          git config --global user.email ${{ secrets.BUILD_USER_EMAIL }}
+          git commit -m "Checksum testing"
+          git push --set-upstream origin with-checksum-testing
```

Comment:
IMHO we have the same problem with pushing in github actions like in #1057. We need to find a solution for this problem and apply it to both actions.